playbooks/maas-tigkstack-influxdb.yml

```diff
@@ -0,0 +1,114 @@
+---
+# Copyright 2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Deploy influxdb
+  hosts: log_hosts
+  gather_facts: true
+  pre_tasks:
+    - name: Check init system
```

Comment:
Please use ansible fact ``{{ ansible_service_mgr }}`` instead of building your own.